ci-operator/templates/cluster-launch-installer-e2e-smoke.yaml

```diff
@@ -202,9 +202,10 @@ objects:
         cd /tmp/cluster
         tectonic init --config=inputs.yaml
         mv -f ${NAME}/* /tmp/cluster/
-        cp config.yaml internal.yaml terraform.tfvars /tmp/artifacts/installer/
+        cp config.yaml internal.yaml /tmp/artifacts/installer/
```

Comment:
It would be nice to get these even if `install` fails.  Maybe put it in an EXIT trap like [this][1] to copy this asset out regardless of `install` success?

[1]: https://github.com/openshift/installer/blob/dc4764dc603cea5da0e54f575b7ae1a2c26d3102/tests/run.sh#L110

Reply:
I'd rather we keep this dead simple since it's holding up some PRs that have to go in. Once those merge, I'm going to come back and update all of this to work with the new installer.